bblfsh/__main__.py

```diff
@@ -1,6 +1,8 @@
 import argparse
 import sys
 
+from bblfsh._pyuast import find
```

Comment:
I don't know if there is specified in PEP8 but there is an old convention in Python of using a leading underline for binary modules.